src/index.test.ts

```diff
@@ -1498,12 +1505,81 @@ describe('Running @erc725/erc725.js tests...', () => {
         );
       });
 
+      it('should decode ALL_PERMISSIONS', () => {
+        const permissions = { ALL_PERMISSIONS: true };
+        const encoded = encodePermissions(permissions);
+
+        const decodedPermissions = decodePermissions(encoded);
+
+        assert.strictEqual(
+          decodedPermissions.ALL_PERMISSIONS,
+          true,
+          'Decoded permissions includes ALL_PERMISSIONS',
+        );
+      });
+
+      it('should not decode CALL or ALL_PERMISSIONS if perms are missing', () => {
+        const permissions = { ALL_PERMISSIONS: true, CALL: false };
```

Comment:
@skimaharvey was not the intended logic to be "all permissions except `CALL`" in this kind of case?

Reply:
yes that's the case. All good